src/avk.cpp

```diff
@@ -7259,10 +7259,11 @@ namespace avk
 				lSrcHandle = aSrcImage->handle(),
 				lDstHandle = aDstImage->handle(),
 				aSrcImageLayout, aDstImageLayout, aImageAspectFlags, aFilter,
-				lExtent = vk::Extent3D{ aSrcImage->width(), aSrcImage->height(), 1u }
+				lExtent = vk::Extent3D{ aSrcImage->width(), aSrcImage->height(), 1u },
+				dExtent = vk::Extent3D{ aDstImage->width(), aDstImage->height(), 1u }

```

Comment:
```suggestion
				lSrcExtent = vk::Extent3D{ aSrcImage->width(), aSrcImage->height(), 1u },
				lDstExtent = vk::Extent3D{ aDstImage->width(), aDstImage->height(), 1u }

```

The `l` prefix is supposed to mean "lambda"